src/components/file/File.js

```diff
@@ -808,6 +844,10 @@ export default class FileComponent extends BaseComponent {
               }
               files.push(fileInfo);
               this.setValue(this.dataValue);
+              // Track uploads in progress.
+              uploadsInProgress.remove(this);
```

Comment:
what if ```uploadsInProgress``` is null here?

Reply:
Shouldn't happen. (Same reason as above.)